core/src/main/scala/scalaz/ImmutableArray.scala

```diff
@@ -15,14 +15,17 @@ import syntax.Ops
 sealed abstract class ImmutableArray[+A] {
   protected[this] def elemTag: ClassTag[A]
 
+  @deprecated("7.2.19", "removed in scalaz 7.3: not total")
```

Comment:
... I think these `@deprecated` args are the wrong way around...

```scala
class deprecated(message: String = "", since: String = "") extends scala.annotation.StaticAnnotation
```